src/submission/exceptions/exceptions.ts

```diff
@@ -335,3 +336,39 @@ export async function getExceptionManifestRecords(
 
 	return exceptionManifest;
 }
+
+/**
+ * Boolean check for existence of exceptions for a specific field
+ *
+ * @param record
+ * @param field
+ * @param schema
+ * @returns boolean indicating if an exception exists
+ */
+export const checkForExceptions = async (
+	record: DeepReadonly<SubmittedClinicalRecord>,
+	field: string,
+	schema: string,
+): Promise<boolean> => {
+	const programId = record['program_id'] as string;
```

Comment:
Error handling if `program_id` is `undefined`? 
`SubmittedClinicalRecord` type doesn't enforce that key so I'm wondering if there's potential for error here

Reply:
added handling